actionview/lib/action_view/helpers/tags/hidden_field.rb

```diff
@@ -5,7 +5,7 @@ module Helpers
     module Tags # :nodoc:
       class HiddenField < TextField # :nodoc:
         def render
-          @options[:autocomplete] = "off"
+          @options[:autocomplete] ||= "off"
```

Comment:
Would this assign `"off"` if the tag were rendered with `autocomplete: false`?

If `"off"` should only ever supersede `nil`, would [Hash#reverse_merge!](https://edgeapi.rubyonrails.org/classes/Hash.html#method-i-reverse_merge-21) work here?

Reply:
`autocomplete: false` seems to generate `autocomplete="false"` rather than remove the attribute. I guess it should be `autocomplete: nil` to remove the default?

Reply:
I've used `reverse_merge!` here since `with_defaults!` makes less sense.